configs/data/im2im/multidim.yaml

```diff
@@ -0,0 +1,54 @@
+train_dataloaders:
+  _target_: monai.data.DataLoader
+  num_workers: 8
+  batch_size: 1
+  pin_memory: True
+  persistent_workers: False
+  shuffle: True
+  dataset:
+    _target_: cyto_dl.datamodules.multidim_image.MultiDimImageDataset
+    # number of workers to use for caching initial dataset
+    num_workers: 8
+    dict_meta:
+      path:
+        - /path/to/your/multidim_image.zarr
```

Comment:
this points to a single image path? you can choose any because youre interested in metadata?